src/linker/Linker/LinkContext.cs

```diff
@@ -308,7 +308,18 @@ public void RegisterAssembly (AssemblyDefinition assembly)
 		{
 			if (SeenFirstTime (assembly)) {
 				SafeReadSymbols (assembly);
-				Annotations.SetAction (assembly, CalculateAssemblyAction (assembly));
+				Annotations.SetAction (assembly,
+					IsCPPCLIAssembly (assembly.MainModule) ? AssemblyAction.Copy : CalculateAssemblyAction (assembly));
```

Comment:
I think this logic needs more tweaks. Do we really want to change the action all the time? What if someone sets "skip" action for the assembly?

I think we should downgrade to copy only for action we cannot handle and issue a warning that about the limitation.

Reply:
> I think we should downgrade to copy only for action we cannot handle and issue a warning that about the limitation.

I agree with the first part - if somebody sets `skip` it should stick, but we should change `link` and `copyused` to `copy`.
I don't think we should warn always - the way it works currently, the SDK relies on linker doing the right thing, so we would be warning basically by default.
Maybe we should warn only if the action was specified explicitly for the assembly, but not warn if we use the default action (if that's possible).

Reply:
I think it'd be a good idea to warn if we are changing the action from an explicit action passed on the command-line.

Reply:
Added a warn for the case where a non-skip/copy action was specified.